src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -542,9 +542,9 @@ private bool SupportsPaging(FirstNameNode node)
             Contracts.AssertValue(node);
 
             var info = GetInfo(node).VerifyValue();
-            if (info.Kind == BindKind.Data &&
-                info.Data is IExternalDataSource dataSourceInfo
-                && dataSourceInfo.IsPageable)
+            if ((info.Kind == BindKind.Data || info.Kind == BindKind.PowerFxResolvedObject) 
```

Comment:
do we need to check the bindkind?

Reply:
TBH no, I don't think there is IsPageable on any other interface or class.